lfs/gitscanner_catfilebatchcheckscanner_test.go

```diff
@@ -0,0 +1,60 @@
+package lfs
+
+import (
+	"bufio"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCatFileBatchCheckScannerWithValidOutput(t *testing.T) {
+	lines := []string{
+		"short line",
```

Comment:
Digging the `assertNextOID` helper func. I think that combined with the contents of each line ("not-a-size", "short-line", etc) is descriptive enough 👍